pkg/maps/nat/nat.go

```diff
@@ -99,6 +115,39 @@ func NewMap(name string, family IPFamily, entries int) *Map {
 	}
 }
 
+type RetriesKey struct {
+	Key uint32
+}
+
+func (k *RetriesKey) String() string  { return fmt.Sprintf("%d", k.Key) }
+func (k *RetriesKey) New() bpf.MapKey { return &RetriesKey{} }
+
+type RetriesValue struct {
+	Value uint32
+}
+
+type RetriesValues []RetriesValue
+
+func (k *RetriesValue) String() string    { return fmt.Sprintf("%d", k.Value) }
+func (k *RetriesValue) New() bpf.MapValue { return &RetriesValue{} }
+func (k *RetriesValue) NewSlice() any     { return &RetriesValues{} }
```

Comment:
Actually, one question, shouldn't this return a pointer to a slice type?

Reply:
Oh I misread, didn't realize RetriesValues was actually an alias for a slice type.

looks good